services_and_applications/jupyter/overview.rst

```diff
@@ -185,6 +188,23 @@ You may follow these general steps to create and install packages within an any
  - *install packages with conda or pip in the activate environment*
 
 
+Example: Creating a Conda environment for Matplotlib Widgets
+============================================================
+
+We currently install Jupyterlab version 2, which needs additional treatment to get interactive extensions
+`such as matplotlib widgets installed <https://github.com/matplotlib/ipympl>`__.
+
+Here is an example how to install matplotlib:
+
+.. code-block:: bash
+
+   conda install -c conda-forge ipympl==0.5.8
+   jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-matplotlib@0.7.3
+
+Once the installation fishes, restart the kernel of your current notebook and refresh the page.
+You can now use the ``%matplotlib ipympl`` magic before importing matplotlib to render interactive plots.
```

Comment:
Note: active in-support ticket.

After this when running in a non-persistent session, I still get
```
Error displaying widget: model not found
```

For a simple test
```py
%matplotlib ipympl
import matplotlib.pyplot as plt
plt.figure()
plt.show()
```

I think to remember that restarting the kernel is not sufficient, but that jupyter needs to be restarted as well... (which would reset the temporary session)

Reply:
This works now by default.